src/main/scala/org/broadinstitute/dsde/agora/server/business/AgoraBusiness.scala

```diff
@@ -505,17 +506,16 @@ class AgoraBusiness(permissionsDataSource: PermissionsDataSource) extends LazyLo
     * @param username user performing the search
     * @return Future containing entity search results.
     */
+  /*
   private def findByMySqlFirst(agoraSearch: AgoraEntity,
```

Comment:
Why not remove the commented code here and above?

Reply:
Removed